### PR TITLE
Allow rails5

### PR DESCRIPTION
**Pull request by seanwalbran**
Update prefactory to work with rails5.

### CODE REVIEW

File: prefactory.gemspec [hunk 1]
```diff
@@ -42,6 +42,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'mysql2', '~> 0.3'
   spec.add_development_dependency 'sqlite3', '~> 1'
   spec.add_development_dependency 'bundler', '~> 1'
-  spec.add_development_dependency 'rake', '~> 10.0'
-  spec.add_development_dependency 'socialcast-git-extensions', '~> 3'
```
Comment (cashins) [inline prefactory.gemspec]: Is this being removed because it isn't rails 5 compatible?

Comment (seanwalbran) [inline prefactory.gemspec]: @cashins Good catch, thanks -- there had been a conflict when I first started working on this but I forgot that I already resolved it in https://github.com/socialcast/socialcast-git-extensions/commit/4fb5a666517ad9874df79590f006d2754439c869#diff-a843bf1d2be4a728a6c3af80efb36f93R19

Comment (seanwalbran) [inline prefactory.gemspec]: Restored in bca4aae

**Comment by cashins** [comment]
+1, thanks!

**Pull request merged**